plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreConfig.java

```diff
@@ -325,4 +326,17 @@ public ThriftMetastoreConfig setWriteStatisticsThreads(int writeStatisticsThread
         this.writeStatisticsThreads = writeStatisticsThreads;
         return this;
     }
+
+    public boolean isBatchMetadataFetch()
+    {
+        return batchMetadataFetch;
+    }
+
+    @Config("hive.metastore.thrift.batch-fetch.enabled")
+    @ConfigDescription("Disables fetching tables and views from all schemas in a single request")
```

Comment:
s/Disables/Enables

Reply:
ditto

Reply:
Ping

Reply:
Changed

Reply:
I think you changed it in a wrong commit :)